### PR TITLE
Feature/238/avaliable signup after agree terms

### PR DESCRIPTION
**Pull request by Tolik170**
**Signup now avaliable after agree terms checkbox**

image)image

### CODE REVIEW

File: tests/unit/containers/guest-home-page/signup-dialog/SignupDialog.spec.js [hunk 1]
```diff
@@ -54,8 +54,12 @@ describe('Signup dialog test', () => {
   })
 
   it('should show error', () => {
+    const checkbox = screen.getByTestId('checkbox')
```
Comment (tarasSanotskyi) [inline tests/unit/containers/guest-home-page/signup-dialog/SignupDialog.spec.js:57]: try to getByRole or getByLabelText

**Review by tarasSanotskyi — CHANGES_REQUESTED** [review]

File: tests/unit/containers/guest-home-page/signup-dialog/SignupDialog.spec.js (hunk 1 above)
Comment (Tolik170) [reply tests/unit/containers/guest-home-page/signup-dialog/SignupDialog.spec.js]: done.

**Review by rostyslaw27 — APPROVED** [review]

File: src/containers/guest-home-page/signup-form/SignupForm.js [hunk 2]
```diff
@@ -8,11 +9,16 @@ import { style } from './signup-form.style'
 
 const SignupForm = ({ handleSubmit, handleChange, handleBlur, data, errors }) => {
   const { t } = useTranslation()
+  const [buttonDisabled, setButtonDisabled] = useState(true)
   const { inputVisibility: passwordVisibility, showInputText: showPassword } = useInputVisibility(errors.password)
   const { inputVisibility: confirmPasswordVisibility, showInputText: showConfirmPassword } = useInputVisibility(
     errors.confirmPassword
   )
 
+  const handleButtonChange = () => {
```
Comment (maarekua) [inline src/containers/guest-home-page/signup-form/SignupForm.js:18]: What button? I think it should be named "onAgreementChange".

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/guest-home-page/signup-form/SignupForm.js (hunk 2 above)
Comment (Tolik170) [reply src/containers/guest-home-page/signup-form/SignupForm.js]: done.

**Review by maarekua — APPROVED** [review]

**Review by High-Voltaged — APPROVED** [review]

**Pull request merged**